### DISCUSSION TITLE
Economic Stimulus

### DISCUSSION THREAD

**Discussion by BilldarBagdar** [markusschloesser/MackieC4_P3#77] [Show and tell]
I decided to behave in a patriotic and community minded manner and spend Uncle Sam's "covid relief / economic stimulus" money this summer rather than banking it or paying down existing debt.  I have a friend who is a full time guitar player and another who runs a recording studio and both have been hurt by the economic slowdown over the last year.  So, I booked some time with the guitar player in the studio to record.  The session is in the middle of July, and I don't really know what we are going to record yet.  I just had the idea and moved on it.  Now I need to pull some shit together.  I have plenty of "halfway" projects floating around, if you know what I mean, but I need to get serious about preparing to record something I'm going to slap my (fake :)) name on. 

What that means here though, is less time coding, more time "finding bugs", well into July for me.

**Comment by markusschloesser** [comment]
Good luck! And I definitely would like to hear the outcome! :-)

**Reply by BilldarBagdar** [reply]
> ...hear the outcome!

sure.  whatever the outcome, I mean I might not want to publicly drop a release on "band camp" or whatever, but I am always willing to share.  Most people who ask once, don't ask again. :)  I started working with this guitar player last year around this time remotely, but the remote connection has been less than ideal in terms of "productivity".

Actually you can help me a great deal right now by answering a question about some lyrics to an old song I have been revisiting.  When I was in high school pushing past 40 years ago now and studying German, a common phrase in the teenage  lingo of the time was "catch a buzz" meaning have some beers, a smoke, or some form of (relatively minor) intoxication like catching a "love buzz" from an attractive, desirable person (or "Cupid's arrow").  We would ask each other, "Do you want to get together after the game Friday and catch a buzz?", for example.  Being mischievous line-skirting students, we translated "catch a buzz" into the German word-phrase "fang ein blitz" as a bit of "private code" between friends.  I have since learned that phrase is basically utter nonsense in "actual spoken German", but I wrote a song around it.  So my question is this.  Is that phrase actually somehow worse than nonsense?  Would it be offensive?  "fang ein blitz zuzammen, vieleicht?"  I'm ok with nonsense, the English words are pretty nonsensical too. (google tells me it means "catch a lightning bolt together, maybe?" which I like poetically and is pretty close to the original coinage we came up with back in the day (booze, especially "DIY liquor", is sometimes called "white lightning" or "lightning in a bottle")  But, at the same time, I would rather not inadvertently piss off every German speaker who hears it because it's taken as an offense.  (pissing any listeners off due to general lyrical meaninglessness is still fair game though :))

**Reply by markusschloesser** [reply]
just quickly: grammatically correct would be "fang ein**en** Blitz". it's basically nonsense here as well, so no harm, someone British however might be reminded of THE BLITZ

**Reply by BilldarBagdar** [reply]
oooo  Good Point.  That British angle had not occurred to me.  Here in the US, our "NFL Football" (and all levels of amateur) features a very common defensive strategy called "blitzing" and references to some team's "devastating blitz" are freely broadcast on TV uncensored. (IMHO - this name fully derives directly from the "Blitzkrieg war strategy" of quickly overwhelming the opposition with more "attackers" than they have "defenders" and winning engagements as much with "speed" as with "force")  More often than not around here I hear that historic "blitz" referenced with qualification as "the London Blitz". (also around here, I can imagine there are a lot of NFL fans who would hear a reference to "the London Blitz" and ask, "Where does he coach?" (both because people can get very passionate about "football", and because a lot of kids can get through school remaining willfully ignorant in key areas much more easily than they should))   I will need to review and mull over that historic perspective (both sides).  Also, of course, my high school self wasn't that great a student, and the extra syllable in the grammatically correct version means I need to "write a new melody" or something if I really want to resurrect that song.

**Comment by BilldarBagdar** [comment]
I ended up cancelling my studio time at the last minute.  I watched some videos on YouTube and now I'm an expert, LOL.  No, I watched a few videos that opened my eyes to the tragic future for which humans are currently building sturdy foundations.  After my eyes were opened, I couldn't justify dumping more carbon into the atmosphere just to fly a musician out to work with me on a "vanity project".  I still paid everyone involved.  It's not their fault I had a change of heart.  The guitarist in particular really needed the money. So, in that sense I successfully redistributed the "stimulus money".  I just didn't get any "work product" in return.  I've more or less quit eating meat too, certainly beef.  Roger Hallam, Stephen Jenkinson, Chris Hedges, Rupert Read, Dahr Jamal, Zizek, Stan Rushworth, Russel Means, Winona LaDuke, John Trudel, Chief Oren Lyons.  I would like to be "remembered fondly" as one of the few who understood what was coming and tried to help.  Tried to make their terrible future a little easier to live in.

That big flood you all experienced in July hardly got any news coverage here.  But I recent watched a "DW channel" "new story video" about it, and holy cow!  I heard one lady tell the camera, "Yes, they told us the storm could bring a big flood.  But we didn't understand what 7 meters meant."  Another person said, "We were stacking sandbags along the bank.  We had about 40 cm of bags above the water and it didn't seem to be rising, didn't seem to be rising.  ...and then it did start rising. But the water wasn't just coming from the river. It was coming from left, right, behind, everywhere.  It was all we could do to grab the dogs, a few valuables, and go upstairs.  We spent a week or 10 days on the top floor.  Firefighters came by in boats after a day or two, asked if we were ok, and gave us food and water..."

Man, 1 meter is a LOT of moving water, 7 must have been shocking. In all the further images I saw of the damage or will see, I keep thinking back to "7 meters".  What would a 25 foot wall of water have looked like tearing through this or that destroyed town?  People must have been terrified. I hope you didn't personally lose anyone close, and my heart goes out to all victims.  Did Cologne see much if any flooding damage?  It seemed to me like maybe most of the terrible flooding was in "side canyon" tributaries, like the Ahrweiler district, and not much "extreme" flooding along the Rhine itself...

**Reply by markusschloesser** [reply]
Hey, I always intended to reply here, but never did. Apologies for that!
The flood spared us here, but lots of basements were flooded even in my area. And just 30km away people did die.
Hope you are ok and still around? :-)
